src/runtime/runtime_esp32xx.go

```diff
@@ -4,28 +4,11 @@ package runtime
 
 import (
 	"device/esp"
-	"machine"
 	"unsafe"
 )
 
 type timeUnit int64
 
-func putchar(c byte) {
-	machine.Serial.WriteByte(c)
-}
-
-func getchar() byte {
-	for machine.Serial.Buffered() == 0 {
-		Gosched()
-	}
-	v, _ := machine.Serial.ReadByte()
-	return v
-}
-
-func buffered() int {
-	return machine.Serial.Buffered()
-}
-
```

Comment:
Is there a reason for this change?
It seems that esp32 and esp32c3 share the same code.